packages/admin-stories/src/docs/components/Stack/Stack.stories.tsx

```diff
@@ -280,4 +288,24 @@ storiesOf("stories/components/Stack", module)
                 </Stack>
             </div>
         );
+    })
+    .add("StackLink", () => {
```

Comment:
I would even say we should promote StackLink as the default solution, also meaning all existing examples should be ported to it. (but that can happen in it's own PR)

Reply:
Okay, I'll create an extra PR for that